index.bs

```diff
@@ -2313,6 +2313,20 @@ The following sections present a set of currently-defined and registered attesta
 The up-to-date list of registered WebAuthn Extensions is maintained in the IANA "WebAuthn Attestation Statement Format
 Identifier" registry established by [[!WebAuthn-Registries]].
 
+## Verification Procedure Inputs ## {#attestation-format-verification-procedures}
```

Comment:
I'm thinking we do not need this new section since it duplicates the info given above in "verification procedure"'s definintion.

Reply:
You're right. I kept the list-style definition but moved it to the site of the [Verification procedure](https://s3.amazonaws.com/pr-preview/emlun/webauthn/issue-560.html#verification-procedure) definition. Does that look good or should I cut it completely?

Reply:
looks good!